packages/react/src/lib/accessibility/Behaviors/Toolbar/toolbarItemBehavior.ts

```diff
@@ -22,7 +22,10 @@ const toolbarItemBehavior: Accessibility<ToolbarItemBehaviorProps> = props => {
   behaviorData.keyActions.wrapper = {
     ...behaviorData.keyActions.wrapper,
     closeMenuAndFocusTrigger: {
-      keyCombinations: props.menu && props.menuOpen ? [{ keyCode: keyboardKey.Escape }] : null,
+      keyCombinations:
+        props.menu && props.menuOpen
+          ? [{ keyCode: keyboardKey.Escape }, { keyCode: keyboardKey.Tab, shiftKey: true }]
```

Comment:
handling `shift+tab` key, so when user press it, focus will go to the prev tabbable element before toolbar, not the trigger button of the menu inside toolbar

Reply:
Just to be sure, `tab` should go to the next focusable element after the toolbar, and `shift+tab` should go to the prev focusable element before the toolbar? `Escape` should focus the `trigger`?

Reply:
yes, it all works. Why only `shift+tab`? - cause trigger is rendered before the menu and trigger has `tabindex=0` when menu is opened. So by default when pressing `shift+tab` - focus goes to to trigger from menu. But need to be at the previous tabbable element